ktor-client/ktor-client-core/common/src/io/ktor/client/features/HttpRedirect.kt

```diff
@@ -47,6 +49,11 @@ public class HttpRedirect {
     public companion object Feature : HttpClientFeature<HttpRedirect, HttpRedirect> {
         override val key: AttributeKey<HttpRedirect> = AttributeKey("HttpRedirect")
 
+        /**
+         * Happens when received response with redirect message.
+         */
+        public val Event: EventDefinition<HttpResponse> = EventDefinition()
```

Comment:
Naming is not consistent. Here it is `HttpRedirect.Event` and below it is `HttpCache.HttpResponseFromCache`. Please use full name here or short below.

Reply:
Sure